commands/project/deploy.ts

```diff
@@ -53,20 +63,22 @@ const validateBuildId = (
       buildId: buildId,
       linkToProject: uiLink(
         i18n(`${i18nKey}.errors.viewProjectsBuilds`),
-        getProjectDetailUrl(projectName, accountId)
+        getProjectDetailUrl(projectName!, accountId)!
       ),
     });
   }
   return true;
-};
+}
 
-exports.handler = async options => {
-  const { derivedAccountId } = options;
+export async function handler(
+  args: ArgumentsCamelCase<ProjectDeployArgs>
+): Promise<void> {
+  const { derivedAccountId } = args;
   const accountConfig = getAccountConfig(derivedAccountId);
-  const { project: projectOption, buildId: buildIdOption } = options;
+  const { project: projectOption, buildId: buildIdOption } = args;
   const accountType = accountConfig && accountConfig.accountType;
 
-  trackCommandUsage('project-deploy', { type: accountType }, derivedAccountId);
+  trackCommandUsage('project-deploy', { type: accountType! }, derivedAccountId);
```

Comment:
`accountType` could still be undefined - not sure if this will affect tracking but would it be better to just pass in `undefined` instead of this object if `accountType` is undeinfed?